app/components/Form/ImageUploadField.js

```diff
@@ -28,12 +29,13 @@ class ImageUploadField extends Component<Props> {
   };
 
   render() {
-    const { className, style, ...props } = this.props;
+    const { className, name, style, ...props } = this.props;
 
     return (
       <div
         className={cx(styles.base, styles.coverImage, className && className)}
```

Comment:
In the `EditorField` file the className can be passed right in, but here we check `classname && className`. The props is optional in both component, so it this unnessesery to have here?
```sh
# Form/EditorField.js
<Editor
className={cx(styles.input, className)}        
{...props}
{...props.input}
{...props.meta}
/>
```

Reply:
Wuut. Yeah

`className && className` is the same as just `className`